src/swap/types.ts

```diff
@@ -141,7 +142,8 @@ export type SwapContextType = {
   from: SwapUnit;
   to: SwapUnit;
   loading: boolean;
-  handleSubmit: (f: (swapTransaction: BuildSwapTransaction) => void) => void;
+  pendingTransaction: boolean;
```

Comment:
Boolean should always start with a prefix like `is` or `has`.

so in this case we want `isPendingTransaction`.

Btw is there a chance where something like `transactionStatus` is a better experience? and `pending` is just a status in it?

Reply:
changed to `isTransactionPending`